packages/react/src/Hero/Hero.module.css

```diff
@@ -63,6 +63,11 @@
   max-width: 600px;
 }
 
+.Hero-description:not(.Hero-description--custom-size) {
```

Comment:
I'm not a big fan of this. This selector increases specificity to override the default size set by the underlying `Text` component. Any other ideas for setting a default size _only when no `size` is provided_?